tiledb/sm/buffer/buffer.cc

```diff
@@ -41,9 +42,9 @@ using namespace tiledb::common;
 namespace tiledb {
 namespace sm {
 
-// =======================================================
-//      BufferBase
-// =======================================================
+//───────────────────────────────────────────────────────
+// BufferBase
+//───────────────────────────────────────────────────────
```

Comment:
turns out my editor of choice does not present these as you intend (reads file, but guess it's displaying single-byte values.)

Reply:
What editor is it?

Reply:
@eric-hughes-tiledb  / @dhoke4tdb is this still a concern / blocking or shall we merge the PR?

Reply:
I will not oppose if those with power to merge accept it
(though I would personally prefer to have it the old way.)

Reply:
I think we should revert to the old format for consistency and open a story about fixing these in the entire code once we decide upon the preferred format.